README.md

```diff
@@ -52,11 +52,7 @@ This module works on macOS and Linux.
 
 ### Databricks provider installation
 
-The module is using the [Databricks Terraform provider](https://github.com/databrickslabs/terraform-provider-databricks). This provider is not in the registry yet and would have to be installed manually. This can be done with the command below:
-
-```sh
-curl https://raw.githubusercontent.com/databrickslabs/terraform-provider-databricks/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins
-```
+The module is using the [Databricks Terraform provider](https://github.com/databrickslabs/terraform-provider-databricks). This provider is in available from the Terraform registry.
```

Comment:
Typo: is in

Reply:
fixed the typo. A version bump to TF 0.13 is necessary to use the module from the tf registry. therefore I have changed the terraform.yml to use 0.13.0 in the testpipeline.